DATA/SHIPS/shiparch.ini

```diff
@@ -14,6 +14,7 @@
 ; \bExternal Mounts\B
 ; Shields:
 ; Thrusters:
+; Mining Device:
```

Comment:
This is just an incomplete solution. Now you added this to every ship's statistics. So if you go for this, _every_ ship must now have a stats entry for Mining Devices.
Or make a copy of this stats entry for just those ships that have actual mining devices.